cmd/osquery-perf/agent.go

```diff
@@ -280,6 +282,26 @@ func (a *agent) config() {
 		return
 	}
 
+	parsedResp := make(map[string]interface{})
+	if err := json.Unmarshal(res.Body(), &parsedResp); err != nil {
+		log.Println("json parse at config:", err)
+		return
+	}
+
+	var scheduledQueries []string
+	for key, val := range parsedResp {
+		if key == "packs" {
+			packDict := val.(map[string]interface{})
```

Comment:
You could alternatively unmarshal to a `fleet.Packs`, right?
https://github.com/fleetdm/fleet/blob/3a973af16418b1ac1ae105c6789f0a49d875e606/server/service/service_osquery.go#L324-L330

Reply:
I think it would be PackSpec.

Reply:
Oh, you meant Pack_s_ plural, nice.